README.md

```diff
@@ -1005,6 +1005,7 @@ _See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <http
 
 ### Video Streaming
 
+- [MediaCMS](https://mediacms.io) - MediaCMS is a modern, fully featured open source video and media CMS, written in Python/Django and React, featuring a REST API. ([Source Code](https://github.com/mediacms-io/mediacms)) `AGPL-3.0` `Python` `Django` `React`
```

Comment:
- Please insert MediaCMS in the alphabetical order
- please set the language/platform to `Python/Docker`

Reply:
Done, hope it's ok!